src/dyad/core/dyad_ctx.c

```diff
@@ -286,24 +289,25 @@ dyad_rc_t dyad_init (bool debug,
     }
 
     ctx->relative_to_managed_path = relative_to_managed_path;
-    DYAD_LOG_INFO (ctx, "DYAD_CORE INIT: relative_to_managed_path %s", \
+    DYAD_LOG_INFO (ctx,
```

Comment:
This will end up in a compilation error.

Reply:
I updated the formatting changes